src/pipeline_v3/core/postgres_fingerprint.py

```
@@ -0,0 +1,412 @@
+                content_hash = EXCLUDED.content_hash,
+                size = EXCLUDED.size,
+                modified_time = EXCLUDED.modified_time,
+                metadata_hash = EXCLUDED.metadata_hash,
+                doc_id = EXCLUDED.doc_id,
+                processing_status = EXCLUDED.processing_status,
+                last_seen = NOW()
+        """
+
+        try:
+            # Build metadata dict
+            metadata = {
```

Ruff (0.11.9)

Local variable `metadata` is assigned to but never used

Remove assignment to unused variable `metadata`

(F841)